.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,40 @@
+name: Feature request
+description: Suggest a new feature for MSAL Python.
+labels: ["feature request", "untriaged", "needs attention"]
+title : '[Feature Request]     '
```

Comment:
This file shall probably use an extension name `.yaml`, rather than `.md`.

Also, a minor suggestion below.

```suggestion
title : '[Feature Request] '
```

Let me know if you want me to make those changes in your branch and then merge this.

Reply:
Sounds good. Main point is to have those labes pre-applied to every new issue / feature request, so as to be able to track them.